tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Validation/CreateOrderItemModelValidatorTests.cs

```diff
@@ -421,9 +421,28 @@ static CreateOrderItemModelValidatorTests()
         }
 
         [Test]
-        [CommonInlineAutoData(nameof(CatalogueItemType.AdditionalService))]
+        [AutoData]
+        public static void Validate_DeliveryDateIsValid_IsAdditionalService_DoesNotHaveError(
+            DateTime deliveryDate,
+            CreateOrderItemModel model,
+            CreateOrderItemModelValidator validator)
+        {
+            var recipient = new OrderItemRecipientModel { DeliveryDate = deliveryDate };
+            model.CatalogueItemType = nameof(CatalogueItemType.AdditionalService);
+            model.ServiceRecipients = new List<OrderItemRecipientModel> { recipient };
+
+            var expectedPropertyName =
+                $"{nameof(CreateOrderItemModel.ServiceRecipients)}[0].{nameof(OrderItemRecipientModel.DeliveryDate)}";
+
+            var result = validator.Validate(model);
+            var failure = result.Errors.FirstOrDefault(e => e.PropertyName == expectedPropertyName);
+
+            failure.Should().BeNull();
+        }
+
+        [Test]
         [CommonInlineAutoData(nameof(CatalogueItemType.AssociatedService))]
```

Comment:
It's probably not worth using inline data here anymore as we only have the one test case now.

Reply:
I've Added in the IsNull_HasError test and updated the Associated service one to use AutoData not inline now